cristin-import/src/main/java/no/unit/nva/cristin/mapper/Nsd.java

```diff
@@ -22,26 +24,55 @@ public Nsd(int nsdCode, int year, ChannelRegistryMapper channelRegistryMapper) {
         this.channelRegistryMapper = channelRegistryMapper;
     }
 
-    public URI createJournalOrSeriesUri() {
-        return lookupNsdJournalOrSeriesProxyUri()
-                   .orElseGet(() -> lookupNsdPublisherProxyUri()
-                                        .orElseThrow(() -> new ChannelRegistryException(nsdCode)));
-    }
-
```

Comment:
Splitting this in two separate methods, one will be used for publicationContext types which can have journals, other second one for series. PublicationContext will never have both.